tests/conformance/testdata/Dockerfile.leadingarg

```diff
@@ -0,0 +1,4 @@
+ARG VERSION latest
```

Comment:
need an '=' between VERSION and latest

Reply:
Oh my… didn't commit my latest change for this file :sweat_smile: :man_facepalming: